ddl/db_integration_test.go

```diff
@@ -527,24 +527,40 @@ func (s *testIntegrationSuite) TestNullGeneratedColumn(c *C) {
 func (s *testIntegrationSuite) TestChangingCharsetToUtf8(c *C) {
 	tk := testkit.NewTestKit(c, s.store)
 
-	tk.MustExec("USE test")
+	tk.MustExec("use test")
+	tk.MustExec("create table t1(a char(10) charset utf8)")
```

Comment:
Could we insert some records into this table and select the records after changing the charset?

Reply:
Okey.